authors/apps/articles/tests/test_comments.py

```diff
@@ -0,0 +1,179 @@
+import json
+from django.urls import reverse
+from rest_framework.views import status
+from rest_framework.test import APITestCase, APIClient
+
+
+class CommentsTestCase(APITestCase):
+    def setUp(self):
+        self.client = APIClient()
+        self.login_url = reverse('authentication:auth-login')
```

Comment:
Kindly remove this, you are not using it anywhere.

Reply:
Done